content/docs/user-guide/data-management/remote-storage/azure-blob-storage.md

```diff
@@ -195,6 +195,26 @@ from an [Azure CLI configuration file] (typically managed with [az config]):
   https://learn.microsoft.com/en-us/cli/azure/azure-cli-configuration#cli-configuration-file
 [az config]: https://docs.microsoft.com/en-us/cli/azure/config
 
+## Configure Azure Remote with environment variables
```

Comment:
This should go to the `More configuration parameters` section.

Let's please move it there. Please keep the same style as we have for other params.

Reply:
I've removed sample configuration as well, is that okay?